### DISCUSSION TITLE
Webhook content on documentation

### DISCUSSION THREAD

**Discussion by willywongi** [directus/directus#11585] [Archive]
I really looked back and forth in the documentation but I can't find information on the actual body content that gets into the webhooks. For now I'm exploring by making actual actions on the collection items and logging it to a custom endpoint - but that's far from ideal. Thank you!

**Comment by rijkvanzanten** [comment]
Good one! We're actually currently working on a big update regarding webhooks that will make the body configurable, so hopefully this documentation won't be needed anymore soon 😁 (cc @erondpowell)

**Comment by willywongi** [comment]
Great! But please let documentation follow the really good standard <3 you have on other features!

**Reply by rijkvanzanten** [reply]
Certainly! Our friend @erondpowell is working tirelessly on upgrading the docs from top to bottom 💪🏻 You can already see the fruits of his labor on [the Insights docs](https://docs.directus.io/app/insights/), and there are [way more goodies to come soon](https://github.com/directus/directus/pulls?q=is%3Apr+author%3Aerondpowell) 😄✨

**Reply by erondpowell** [reply]
@willywongi Please let me know if you have any thoughts, ideas, suggestions, or feedback!

I have a strategy in place for the docs to try and make them "follow the really good standard" as you worded it, but I'm very curious what the community is looking for in documentation.

**Reply by willywongi** [reply]
- A short and complete "getting started"
- Coherent naming (every "thing" has the same name through the pages)
- Easy to dig deep if you need to understand better some concept
- Easy to find definitions of every resource
- Rich in examples
- Explicit 

As an example: I love that every resource in the documentation has an explicit "Supports all global query parameters." with a link to the reference of global query parameters. It would be of no use writing once "All resources except foo and bar support global query parameters", because I will never read all the reference start-to-finish, but I will jump across different pages.

I'm available for more chat on this topic, if needed! 
Grazie 🙏🏻 , ciao 👋🏻 !

**Reply by erondpowell** [reply]
Awesome suggestions! @willywongi you might be happy to know that all this and more is in the pipelines, though some points will take quite a while to execute properly. I would really love to chat more on these topics when you have time- maybe at some point in the next month or two. Are you in the Discord community yet?

**Reply by willywongi** [reply]
Yes, I am now!

**Reply by erondpowell** [reply]
Sweet, I just tried to DM you
